lib/matplotlib/tests/test_collections.py

```diff
@@ -727,7 +727,7 @@ def test_pathcollection_legend_elements():
 
     levels = [-1, 0, 55.4, 260]
     h6, lab6 = sc.legend_elements(num=levels, prop="sizes", fmt="{x:g}")
-    assert_array_equal(np.array(lab6).astype(float), levels[2:])
+    assert_array_equal(np.asarray(lab6, float), levels[2:])
```

Comment:
How come this one became `np.asarray` when the above one remained `np.array`?

Reply:
Switched to a plain assert, as above.